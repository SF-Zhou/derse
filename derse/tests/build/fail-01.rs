#[derive(derse::Serialize)]
union T1 {}

fn main() {}
