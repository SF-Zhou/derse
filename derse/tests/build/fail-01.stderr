error: proc-macro derive panicked
 --> tests/build/fail-01.rs:1:10
  |
1 | #[derive(derse::Serialize)]
  |          ^^^^^^^^^^^^^^^^
  |
  = help: message: only struct and enum are supported

error: unions cannot have zero fields
 --> tests/build/fail-01.rs:2:1
  |
2 | union T1 {}
  | ^^^^^^^^^^^
